en/reference/nativerank.html

```diff
@@ -659,8 +659,12 @@ <h2 id="nativeProximity">nativeProximity</h2>
 <p>
   In this case only query term pairs searching in index fields <em>f1</em> and <em>f2</em> are considered.
 </p>
-
-
+<p>
+  Consider an array of string field with three elements <code>["a c", "x b x a x x x x x x b", "x x d"]</code>.
```

Comment:
Would this benefit from saying "Consider an `array<string>` field (...)" since that's what it would look like in the schema?